docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       db:
         condition: service_healthy
   db:
-    image: neo4j:3.4-enterprise
+    image: neo4j:enterprise
```

Comment:
Can we stay on the latest 3.x for now please? This matches the current prod environment.

Reply:
Sure. Although, I do want to check if the newer release has some new capabilities which can be useful.
BTW, where is the code which deploys the db tracked?

Reply:
Done